### PR TITLE
Add secret resource of type ssh key

### PR DESCRIPTION
**Pull request by rathodmeetsatish**
Add secret resource of type ssh key

### CODE REVIEW

File: internal/service/platform/secret/sshkey.go [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+				"tgt_key_tab_file_path_spec": readTgtKeyFilePathSpec(secret),
+				"tgt_password_spec":          readTgtPasswordSpec(secret),
+			},
+		})
+
+	}
+	return nil
+}
+
+func readSshKeyRefernceCredentialSpec(secret *nextgen.Secret) []map[string]interface{} {
```
Comment (rijajoo) [inline internal/service/platform/secret/sshkey.go:362]: Typo

File: internal/service/platform/workspace.code-workspace [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (rijajoo) [inline internal/service/platform/workspace.code-workspace:1]: This is not needed right?

File: internal/service/platform/secret/sshkey.go (hunk 1 above)
Comment (rathodmeetsatish) [reply internal/service/platform/secret/sshkey.go]: Done

File: internal/service/platform/workspace.code-workspace (hunk 2 above)
Comment (rathodmeetsatish) [reply internal/service/platform/workspace.code-workspace:1]: yes

**Review by rijajoo — APPROVED** [review]

**Pull request merged**